.github/workflows/publishTypes.yml

```diff
@@ -0,0 +1,63 @@
+# Very important!
+# Make sure that the github token has read AND WRITE access on github.
+#   1. hit https://github.com/seatsio/[REPO]/settings/actions
+#   2. under "Workflow permissions", make sure "Read and write permissions" is checked instead of the (default?) read only.
+
+name: 'Update and publish @seatsio/seatsio-types'
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Bump version or release current version without changing package.json. Major for incompatible API changes, minor for adding backwards compatible features, patch for bugfixes.'
+        required: true
+        default: 'patch'
+        type: choice
+        options:
+          - current
+          - patch
+          - minor
+          - major
+
+jobs:
+  updateVersionAndPublish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: fregante/setup-git-user@v2
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+          registry-url: https://registry.npmjs.org/
+
+      - name: Build types
+        working-directory: ./ui/seatsio-types
+        run: |
+          yarn install
+          yarn build
+        
+      - name: Checkout latest master
+        working-directory: ./ui/seatsio-types
```

Comment:
We'll need to fix those references to the ui folder

Reply:
Changed to work on root folder. License file is already present on master (only file there, MIT license).